src/test/java/com/answerdigital/answerking/service/OrderServiceTest.java

```diff
@@ -81,109 +79,131 @@ void testAddOrderWithNoProductsValidOrderRequestIsSuccessful() {
+        final OrderResponse response = orderService.getOrderResponseById(order.getId());
+
+        // then
+        assertAll("Should be equal",
+            () -> assertEquals(order.getId(), response.getId()),
+            () -> assertEquals(order.getLastUpdated(), response.getLastUpdated()),
+            () -> assertEquals(order.getCreatedOn(), response.getCreatedOn()),
+            () -> assertEquals(order.getOrderStatus(), response.getOrderStatus())
+        );
         verify(orderRepository).findById(anyLong());
     }
 
     @Test
-    void testFindByIdThrowsNotFoundException() {
-        // When
-        when(orderRepository.findById(anyLong()))
-                .thenReturn(Optional.empty());
+    void testGetOrderResponseByIdWithInvalidIdThrowsNotFoundException() {
+        // when
+        doReturn(Optional.empty())
+            .when(orderRepository)
+            .findById(anyLong());
 
-        // Then
-        assertThrows(NotFoundException.class,
-                () -> orderService.getOrderResponseById(2L));
+        // then
+        assertThrows(NotFoundException.class, () -> orderService.getOrderResponseById(NONEXISTENT_ORDER_ID));
         verify(orderRepository).findById(anyLong());
     }
 
     @Test
     void testFindAllReturnsEmptyListOfOrders() {
-        // Given
+        // given
         final List<Order> orders = List.of();
 
-        // When
-        when(orderRepository.findAll())
-                .thenReturn(orders);
+        // when
+        doReturn(orders)
+            .when(orderRepository)
+            .findAll();
 
         final List<OrderResponse> response = orderService.findAll();
 
-        // Then
+        // then
         assertTrue(response.isEmpty());
         verify(orderRepository).findAll();
     }
 
     @Test
     void testFindAllReturnsListOfOrders() {
-        // Given
+        // given
         final List<Order> orders = List.of(
             orderTestBuilder.withId(1L).build(),
             orderTestBuilder.withId(2L).build()
         );
 
-        // When
+        // when
         doReturn(orders)
             .when(orderRepository)
             .findAll();
 
         final List<OrderResponse> response = orderService.findAll();
 
-        // Then
+        // then
         assertEquals(2, response.size());
         assertFalse(response.isEmpty());
         verify(orderRepository).findAll();
     }
 
     @Test
     void testUpdateOrder() {
-        // Given
-        final Order originalOrder = orderTestBuilder.withDefaultValues().build();
+        // given
+        final Order order = orderTestBuilder
+            .withDefaultValues()
+            .build();
+
         final OrderRequest updateOrderRequest = orderRequestTestBuilder
                 .withLineItemRequests(List.of(new LineItemRequest(1L, 1)))
                 .build();
-        final Order expectedOrder = new Order();
+
         final Product product = Product.builder()
```

Comment:
Can we replace this with a ProductTestbuilder please? This is Lombok's builder and has some magic values.